src/locale/de/dim.json

```diff
@@ -91,7 +91,7 @@
     "ImportNoFile": "Keine Datei ausgewählt.",
     "ImportSuccess": "Wunschliste geladen mit Informationen über einen Roll.{{titleAndDescription}}",
     "ImportSuccess_plural": "Wunschliste mit Informationen über {{count}} Rolls geladen.{{titleAndDescription}}",
-    "Num": "{{count, number}} Rolls in deiner Wunschliste"
+    "Num": "{{number}} Rolls in deiner Wunschliste"
```

Comment:
I was using the `, number` filter to format as a localized number - do we not need that?

Reply:
ah I forgot about the formatting switch case ty